test/data/datasets.js

```diff
@@ -0,0 +1,22 @@
+import faker from 'faker';
+import { comparator } from 'ramda';
+
+import { dataStore } from './data-store';
+import { extendedProjects } from './projects';
+
+// eslint-disable-next-line import/prefer-default-export
+export const extendedDatasets = dataStore({
```

Comment:
I named this `extendedDatasets` for now with the thought that we might end up requesting extended metadata. If we don't end up doing so, we can rename this to `standardDatasets`.